src/ShopBundle/Resources/translations/messages.de.xliff

```diff
@@ -174,44 +174,44 @@
 
             <trans-unit id="chameleon_system_shop.module_basket.error_no_data_sent_to_update_basket_items">
                 <source>chameleon_system_shop.module_basket.error_no_data_sent_to_update_basket_items</source>
-                <target>Es wurden keine Daten an UpdateBasketItems übermittelt</target>
+                <target>Es wurden keine Daten an UpdateBasketItems übermittelt.</target>
             </trans-unit>
             <trans-unit id="chameleon_system_shop.module_basket.error_basket_item_key_unknown">
                 <source>chameleon_system_shop.module_basket.error_basket_item_key_unknown</source>
-                <target>Es gibt kein Produkt mit dem sBasketItemKey %key%</target>
+                <target>Es gibt kein Produkt mit dem sBasketItemKey %key%.</target>
```

Comment:
Is this displayed to the user? If so I don't think this is an appropriate wording. Nobody knows what an `sBasketItemKey` is.

Reply:
My focus was mostly on typos - you're right that it's not a good message, but I'd need to dive deeper to find a good message, which I'd like to avoid at this point.